torch/testing/_internal/common_methods_invocations.py

```diff
@@ -855,9 +855,10 @@ def __init__(
         identity: Optional[Union[int, float, complex]] = None,
 
         # possible values are:
-        # - propagate: NaN values are propagated to the output
-        # - omit: only non-NaN values participate in computing reduction
-        nan_policy: str = 'propagate',
+        # - 'propagate': NaN values are propagated to the output
+        # - 'omit': only non-NaN values participate in computing reduction
+        # - None: operator decides how to handle NaNs.
```

Comment:
Would you elaborate on this addition?

Reply:
None isn't actually used by any of the ReductionInfos, right?

Reply:
Though they could still offer nan_policy if a user for instance wants to compute all of only the non-NaN values.